### ISSUE TITLE
google_compute_project_metadata: sshKeys shows up in response; generates diff 

### ISSUE THREAD

**Issue by phinze** [hashicorp/terraform#4504]
We're getting this acceptance test failure on master:

https://travis-ci.org/hashicorp/terraform/jobs/100228177#L631

```
--- FAIL: TestAccComputeProjectMetadata_basic (27.35s)
	testing.go:144: Step 0 error: After applying this step, the plan was not empty:
		
		DIFF:
		
		UPDATE: google_compute_project_metadata.fizzbuzz
		  metadata.#:       "3" => "2"
		  metadata.sshKeys: "" => ""
		
		STATE:
		
		google_compute_project_metadata.fizzbuzz:
		  ID = common_metadata
		  metadata.# = 3
		  metadata.banana = orange
		  metadata.sofa = darwinism
		  metadata.sshKeys =
```

Not sure what the best action is to take here - should we special case `sshKeys` and ignore that field for project metadata purposes?

Pinging @lwander to find out!

**Comment by lwander** [comment]
Looking into this now!

**Comment by phinze** [comment]
Fixed in #4512, thanks again @lwander!
